src/workloads/scale/LargeIndexedInsMatchingDocuments.yml

```diff
@@ -0,0 +1,84 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This is an indexed $in workload that matches documents in a collection.
+  This workload is intended to test a more representative workload
+
+  An extremelly common usecase for users is to fetch objects and hydrate with follow up $in query
+
+  For example:
+  The first query fetches an Author object
+  {
+    type: "author",
+    name: "Tyler",
+    posts: [ObjectId(1), ObjectId(2), ObjectId(3)],
+  }
+
+  The second query fetches the related, but not embedded/denormalized objects
+
+  { $in: [ObjectId(1), ObjectId(2), ObjectId(3)] }
+
+  This workload focuses on the second query of this use case matching documents
+  using arrays of integers that are generated to be used for the $in queries.
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 20000
+  Update:
+    QueryOptions:
+      maxPoolSize: 100
+
+Actors:
+  - Name: BlogPostsLoader
+    Type: Loader
+    Threads: 1
+    Phases:
+      - Repeat: 1
+        Database: indexedins
+        CollectionCount: 1
+        Threads: 1
+        DocumentCount: 2048000
+        BatchSize: 100
+        Document:
```

Comment:
The documents work out ~57 bytes:

```
Object.bsonsize({_id: 2048000, title: '12345678', text: '12345678'})
```

2,048,000 works out at about 0.1 GB, is this a big enough test?

Reply:
we discussed internally and settled on around 2,000,000 value for the document count. We think it should cover the workload needs. 
We can increase if we notice it's not enough.